lib/commands/wallet/vanity.js

```diff
@@ -10,13 +10,14 @@ const schema = {
   vanity: Joi.string().required()
 }
 
-function __createVanityAddress(netVersion, vanity) {
+function __createVanityAddress (netVersion, vanity) {
   const arkjs = require('arkjs')
   const bip39 = require('bip39')
-  let passphrase, address = ''
+  let passphrase
+  let address = ''
```

Comment:
Unless you do `address +=` you can omit the `= ''`.

Reply:
thx, I imagined it would throw because null.toLowerCase().I'll update it in the next PR.

Reply:
You can just leave things like that in, will do a refactor at the end anyways.

Reply:
just got an error: Cannot read property 'toLowerCase' of undefined :)

I'll put it back in there next PR